modules/olm-installing-opm.adoc

```diff
@@ -2,18 +2,25 @@
 //
 // * operators/olm-managing-custom-catalogs.adoc
 
+ifdef::openshift-origin[]
+:registry-image: quay.io/openshift/origin-operator-registry:4.6.0
+endif::[]
+ifndef::openshift-origin[]
+:registry-image: registry.redhat.io/openshift4/ose-operator-registry:v4.6
```

Comment:
@adellape  the url "registry.redhat.io/openshift4/ose-operator-registry:v4.6" is correct? I try to pull it, but it return 404 error.

Reply:
I can't test it in the wild until GA, but this is the image that we used for the opm install steps in the 4.5 docs, using a `v4.5` tag.

https://catalog.redhat.com/software/containers/openshift4/ose-operator-registry/5cddd0bed70cc57c44b2e1f3?tag=all

Reply:
I was able to test the OKD version of the steps using the quay.io image, however.

Reply:
@adellape OK, got it. please have a try after GA.